useful-scripts/misc-test-utils/package.json

```diff
@@ -29,6 +29,7 @@
     "typescript": "^4.3.5"
   },
   "dependencies": {
+    "guid-typescript": "1.0.9",
```

Comment:
Would suggest using the `uuid` package (with `@types/uuid`) instead. It's way more popular and is actively maintained unlike the above package.

Reply:
Looks like the `validate` function from UUID doesn't seem to play nice with Microsoft's GUIDs. I attempted to upload a subject using an existing release's ID (with validation in place from the UUID package) but it seems that the validate function throws an error. I initially thought that the UUID package generated UUIDs / GUIDs in a different way to how Microsoft defines GUIDs which is why I chose the `guid-typescript` package. 

```
 >> Not a valid GUID
```